### DISCUSSION TITLE
Disabling idle database unloading

### DISCUSSION THREAD

**Discussion by aventus13** [ravendb/ravendb#12130] [Q&A] [answered]
RavenDB provides an option to [configure the time](https://ravendb.net/docs/article-page/5.0/csharp/server/configuration/database-configuration#databases.maxidletimeinsec) after which an idle database should be unloaded from memory. However is there a way to disable unloading completely? I have a low-traffic database that gets unloaded often. Because of this, visiting the website causes a brief lag while it's waiting for the database server to respond, which doesn't provide a good user experience. 

If such an option is not available, could this be added? For example, setting `Databases.MaxIdleTimeInSec` to `0` should prevent the server from performing idle checks.

By the way- welcome and glad to see GH Discussions available! Hopefully over time people will migrate from Google Groups to here.

**Comment by garayx** [comment, marked as answer]
Hi 

you can set the [Databases.MaxIdleTimeInSec
](https://ravendb.net/docs/article-page/5.0/csharp/server/configuration/database-configuration#databases.maxidletimeinsec) configuration to int.MaxValue (2147483647), this will prevent you databases from idling.
